### ISSUE TITLE
Create page sections

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#69]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [x] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To place widgets into a single-column section and a configurable two-column section.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by JanLenoch** [comment]
Resolved via e98a78ed8e7aaa3879fd584e246f58baf7916c60
